### DISCUSSION TITLE
osu!stream ruleset in osu!lazer

### DISCUSSION THREAD

**Discussion by BKVad1m** [ppy/osu#20325] [Ideas] [answered]
![image](https://user-images.githubusercontent.com/76803825/190432837-d872d4f7-292b-45c8-a68c-3e1c32f933a9.png)
osu!stream celebrating 10th anniversary 
osu!stream ruleset including
- double fingers
- hold note feature
- hit both circles and sliders in same time
- available mobile and arcade
see: https://osu.ppy.sh/wiki/en/osu%21stream

**Comment by peppy** [comment, marked as answer]
stream-like gameplay elements will be added to osu! (i have personal plans to do this).

that said, if you want to play stream, play stream. a ruleset will not be made.
